src/tests/integration/test_string.py

```diff
@@ -128,33 +171,52 @@ def test_cli_rule_to_kast(definition_dir: Path, text: str) -> None:
 
 
 @pytest.mark.parametrize('text', TEST_DATA, ids=TEST_DATA)
-def test_kore_rpc(definition_dir: Path, text: str) -> None:
-    def config(k: Pattern | None, s: Pattern) -> Pattern:
-        dotk = App('dotk', (), ())
-        kseq = App('kseq', (), (App('inj', (STRING, SortApp('SortKItem')), (k,)), dotk)) if k else dotk
-        return App(
-            "Lbl'-LT-'generatedTop'-GT-'",
-            (),
-            (
-                App("Lbl'-LT-'k'-GT-'", (), (kseq,)),
-                App("Lbl'-LT-'generatedCounter'-GT-'", (), (int_dv(0),)),
-                App("Lbl'-LT-'s'-GT-'", (), (s,)),
-            ),
-        )
+def test_krun(haskell_dir: Path, text: str) -> None:
+    # Given
+    kore = kore_config(text, '')
+    expected = kore_config(None, text)
+    krun = KRun(haskell_dir)
```

Comment:
If you want to see this break:

```suggestion
def test_krun(definition_dir: Path, text: str) -> None:
    # Given
    kore = kore_config(text, '')
    expected = kore_config(None, text)
    krun = KRun(definition_dir)
```

Reply:
Can we do something like
```
@pytest.mark.parametrize('text', TEST_DATA, ids=TEST_DATA)
def test_krun(definition_dir: Path, llvm_dir: Path, text: str) -> None:
    if definition_dir == llvm_dir:
        # https://github.com/runtimeverification/k/issues/3344
        pytest.skip()
```
to document that this is failing test which is intentionally skipped?